src/application.ts

```diff
@@ -74,10 +71,35 @@ export class MyriadApiApplication extends BootMixin(
     // initialize firebase app
     firebaseAdmin.initializeApp({
```

Comment:
bor @abdulhakim2902 bagian ini samain aja kek this.bindComponent(); dan this.bindService();
ini perlu check ENV, kalau null kasih gini aja
`firebaseAdmin.initializeApp();`
tapi kalau ada baru kasih options nya